oshi-core/src/main/java/oshi/util/ExecutingCommand.java

```diff
@@ -81,7 +83,16 @@ private ExecutingCommand() {
     public static List<String> runNative(String[] cmdToRunWithArgs) {
         Process p = null;
         try {
-            p = Runtime.getRuntime().exec(cmdToRunWithArgs);
+            PlatformEnum platform = SystemInfo.getCurrentPlatformEnum();
+            if (platform == PlatformEnum.WINDOWS) {
+                p = Runtime.getRuntime().exec(cmdToRunWithArgs, new String[]{"LANGUAGE=C"});
+            } else if (platform == PlatformEnum.LINUX || platform == PlatformEnum.FREEBSD
+                    || platform == PlatformEnum.SOLARIS || platform == PlatformEnum.AIX
+                    || platform == PlatformEnum.MACOSX) {
+                p = Runtime.getRuntime().exec(cmdToRunWithArgs, new String[]{"LC_ALL=C"});
+            } else {
```

Comment:
This `else` case will never happen. It's either Windows or *nix. You can simplify the previous condition to be the "else" without testing the platform.

Reply:
I actually thought about checking for windows and treat everything else as *nix but then found that there is also UNKNOWN value in the enum so wanted to make sure that there are no surprises when new not windows and not unix platform is added but if you say its fine I will change it 👍

Reply:
It is in the enum, but also throws an exception at SystemInfo instantiation.